### DISCUSSION TITLE
Does following Techniques reduce model quality by huge margin ?

### DISCUSSION THREAD

**Discussion by Greatz08** [KellerJordan/modded-nanogpt#48] [General]
i was curious to know about the actual performance of model after using multiple techniques to speed run.

**Comment by BradKML** [comment]
Often times there are two answers to this:
1. Just train it for longer, and the quality issue will generally fade away (generic pedestrian answer)
2. Quality degradation exists (expert-oriented response, maybe @KellerJordan know more about this)

I would like to know which possibility is more likely to be true tho

**Reply by aifartist** [reply]
How does one check the quality if there is no way to do inference with this model?
